sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedContextClientImpl.java

```diff
@@ -22,6 +24,7 @@
 import com.azure.cosmos.implementation.AsyncDocumentClient;
 import com.azure.cosmos.implementation.PartitionKeyRange;
 import com.azure.cosmos.implementation.changefeed.ChangeFeedContextClient;
+import com.azure.cosmos.util.Beta;
```

Comment:
I see we are importing it, but not using it ?

Reply:
yep, added them accidentally; I'll get them fixed.